cmd/hooks/hooks.go

```diff
@@ -823,3 +828,64 @@ func closeWithError(closer io.Closer) {
 		fmt.Printf("%v, an error occurred while closing the client", err)
```

Comment:
Can you help update this line to use the logger if it's in scope?

Reply:
Ok